lib/reducers/sessions.js

```diff
@@ -41,7 +43,8 @@ const reducer = (state = initialState, action) => {
           cols: action.cols,
           rows: action.rows,
           uid: action.uid,
-          shell: action.shell.split('/').pop(),
+          shell: action.shell.split(path.sep).pop(),
+          isWsl: action.isWsl,
```

Comment:
To be sure, are you suggesting I change this to `isWsl: isWslShell(action.shell)`?

Reply:
@chabou There's a little problem with that I'm not sure how to handle given that I don't know the code base so well. The `isWsl` function was created as an `app` util function and this reducer is inside the `lib` folder. I'm not sure I can/should import stuff from `app` into `lib` (and vice-versa for that matter).  How should I proceed?

Reply:
I would suggest to move `app/utils/wsl.js` to `lib/utils/wsl.js` or move your function directly into `lib/reducers/sessions.js` because your function will not be used anywhere else.